core/unit_test/TestWithoutInitializing.hpp

```diff
@@ -97,6 +99,44 @@ TEST(TEST_CATEGORY, resize_realloc_no_alloc) {
   listen_tool_events(Config::DisableAll());
 }
 
+TEST(TEST_CATEGORY, realloc_exec_space) {
+#ifdef KOKKOS_ENABLE_CUDA
+  if (std::is_same<typename TEST_EXECSPACE::memory_space,
+                   Kokkos::CudaUVMSpace>::value)
+    GTEST_SKIP() << "skipping since CudaUVMSpace requires additional fences";
```

Comment:
Does the recently-added `HIPManagedSpace` also require a fence, and hence a skipped test? I only ask because it's described as operating similarly to `CudaUVMSpace`, but its underlying implementation may be more forgiving in this regard.

Reply:
It should not. The allocation done by HIP seems to be immediately accessible

Reply:
No, it doesn't, we decided that those fences shouldn't be necessary.